packages/mantine/src/styles/Skeleton.module.css

```diff
@@ -1,11 +1,11 @@
 .root {
-    &[data-visible] {
+    &:where([data-visible]) {
```

Comment:
I am curious, what is the effect of using [:where](https://developer.mozilla.org/en-US/docs/Web/CSS/:where) here.

Reply:
I wanted to override Mantine's selector so I re-used the same thing https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/core/src/components/Skeleton/Skeleton.module.css#L24